.travis.yml

```diff
@@ -101,3 +109,4 @@ branches:
     - master
     - "1.1"
     - "2.0"
+    - ppc64le
```

Comment:
This might have been useful for testing but we don't want to run travis on additional branches -- these are just for the branches in the central repo.

Reply:
Thanks!! . As per you your suggestion, I have removed -ppc64le from only branch section.